lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -3086,6 +3111,17 @@ struct CanonicalizeConvertFromReshape
     auto convert = op.getSrc().getDefiningOp<ConvertLayoutOp>();
     if (!convert)
       return failure();
+    // If the layouts are structurally the same, the convert is trivial
+    // Should we do this whenever cvtReordersRegisters is true?
```

Comment:
>     // Should we do this whenever cvtReordersRegisters is true?

I don't think so, a convert layout op is still required in that case so you cannot just remove it

Reply:
yep, old comment I wrote in the passing. Removing.